examples/Cargo.toml

```diff
@@ -276,13 +276,13 @@ tracing = ["dep:tracing", "dep:tracing-subscriber"]
 uds = ["tokio-stream/net", "dep:tower", "dep:hyper", "dep:hyper-util"]
 streaming = ["tokio-stream", "dep:h2"]
 mock = ["tokio-stream", "dep:tower", "dep:hyper-util"]
-tower = ["dep:hyper", "dep:hyper-util", "dep:tower", "dep:http"]
+tower = ["dep:hyper", "dep:hyper-util", "dep:tower", "tower?/timeout", "dep:http"]
```

Comment:
Does the previous commit fail to compile the examples? If so, IMO we should squash this commit into it.

Reply:
> Does the previous commit fail to compile the examples? 

Yes. Squashed these two commits.